ui/app/src/modules/Preview/Tools/AssetsPanel.tsx

```diff
@@ -166,14 +169,18 @@ export default function AssetsPanel() {
   const initialKeyword = useSelection((state) => state.preview.assets.query.keywords);
   const [keyword, setKeyword] = useState(initialKeyword);
   const [dragInProgress, setDragInProgress] = useState(false);
+  const [error, setError] = useState(null);
```

Comment:
Error comes from the resource. Not needed in the state at this level. You should remove the Panel error prop